R/misc.R

```diff
@@ -0,0 +1,73 @@
+#' Helper function to print out log messages into Shiny
```

Comment:
I am not a big fan of modules named "misc", "utils", etc. This doesn't give me any additional information when I look for some function. I would suggest to rename it to `logger.R` and keep `log_msg` only, and move `cleanup_hashpath` and `extract_link_name` to the `router.R`